cli/helm/install.go

```diff
@@ -91,15 +91,15 @@ func InstallHelmRelease(options *InstallOptions) error {
 
 	if !options.AutoApprove {
 		confirmation, err := options.UI.Input(&terminal.Input{
-			Prompt: "Proceed with installation? (y/N)",
+			Prompt: "Proceed with installation? (Y/n)",
 			Style:  terminal.InfoStyle,
 			Secret: false,
 		})
 
 		if err != nil {
 			return err
 		}
-		if common.Abort(confirmation) {
+		if confirmation != "" && common.Abort(confirmation) {
```

Comment:
I just suggest adding a comment on the intention here, that we intend to be approving on default